smalllog.txt

```diff
@@ -0,0 +1,2238 @@
+D 21:33:52.1938781 GameState.DebugPrintPowerList() - Count=103
```

Comment:
nit: I think I'd put sample log files in their own directory to keep the root directory small. This is totally a personal preference, and I don't mind if you ignore this comment.